addons/hr_recruitment/models/hr_applicant.py

```diff
@@ -591,6 +591,7 @@ def _get_employee_create_vals(self):
             'work_email': self.department_id.company_id.email or self.email_from, # To have a valid email address by default
             'work_phone': self.department_id.company_id.phone,
             'applicant_id': self.ids,
+            'default_phone': self.partner_phone or self.partner_mobile
```

Comment:
```suggestion
            'phone': self.partner_phone or self.partner_mobile
```

Reply:
@arpi-odoo here we just return dictionary of values to create employee.

 `default_xxx` is doing something else. That we need for actions. Let's say we redirect to a new employee form, then if we have `default_phone` in the action return,  `phone` field will be pre-filled with the value that we give. But that's not what we are doing here